app/src/main/java/org/fossasia/openevent/general/event/EventsViewModel.kt

```diff
@@ -14,29 +14,37 @@ class EventsViewModel(private val eventService: EventService, private val prefer
     private val tokenKey = "LOCATION"
 
     val progress = MutableLiveData<Boolean>()
+    val showSchimmerEvents = MutableLiveData<Boolean>()
     val events = MutableLiveData<List<Event>>()
     val error = MutableLiveData<String>()
 
     val savedLocation by lazy { preference.getString(tokenKey) }
 
+    fun setShowSchimmerEvents(value: Boolean) {
+        showSchimmerEvents.value = value
+    }
```

Comment:
This function is redundant. View should not trigger things in viewmodel. View model should react to view's lifecycle events